.gitignore

```diff
@@ -1,2 +1,4 @@
 test/js/lib/
 jsDAVlib.zip
+.idea/
+*.iml
```

Comment:
Add dist/jsDAVlib.js since it's generated by makefile.
Also remove it of the commit ;)